vendor/engines/bulk_email/app/assets/javascripts/bulk_email/bulk_email.js

```diff
@@ -31,4 +31,15 @@ $(function() {
     return false;
   })
 
+  function showHideRecipientExportButton() {
+    if ($('.js--bulk-email-recipient').is(':checked')) {
+      $('.js--bulk-email-export-button').show();
```

Comment:
What about disabling it so it's grayed out? Then you don't have to worry about the shifting.